static/tests/frontend/specs/commentIconsGroups.js

```diff
@@ -0,0 +1,232 @@
+describe("ep_comments_page - group multiple icons on line", function() {
```

Comment:
Acho que você usou o arquivo `test/frontends/specs/commentIcons.js` como base pra construir esse teste. Certo?

O problema é que esse arquivo é bem antigo e está fora dos padrões que temos trabalhado.
Seguem alguns problemas:

- devemos usar aspas simples nas strings
- as funções auxiliares neste arquivo deveriam estar dentro de uma estrutura `helperFunctions`. Veja [static/tests/frontend/specs/commentCopyPaste.js](https://github.com/storytouch/ep_comments/blob/9c11431053d76f1592206bf91d56b5a5e7099aa8/static/tests/frontend/specs/commentCopyPaste.js) como exemplo.
- evitar o uso the `should ...` nas descrições do it